src/recently-viewed.css

```
@@ -0,0 +1,263 @@
+  gap: 8px;
+  align-items: flex-end;
+  padding-top: 20px;
+}
+
+.recently-viewed-card h4 {
+  font-size: 13px;
+  font-weight: 700;
+  color: var(--ink);
+  line-height: 1.3;
+  margin: 0;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)